testsuite/MDAnalysisTests/coordinates/test_gro2.py

```diff
@@ -0,0 +1,24 @@
+from MDAnalysis.coordinates.GRO import GROReader, GROWriter
+from MDAnalysisTests.coordinates.base import BaseReference, BaseReaderTest
+from MDAnalysisTests.datafiles import COORDINATES_GRO
+import numpy as np
+
+
+class GROReference(BaseReference):
+    def __init__(self):
+        super(GROReference, self).__init__()
+        self.trajectory = COORDINATES_GRO
+        # self.topology = COORDINATES_GRO
+        self.reader = GROReader
+        self.writer = GROWriter
+        self.ext = 'gro'
+        # self.volume = 0
+        # self.dimensions = np.zeros(6)
+        # self.container_format = True
```

Comment:
General comment. The general purpose BaseReaderTest shouldn't know anything about the specific extensions used. You should enable/disable tests based on values in the reference. 

Why doesn't the skipif work?